ecs-cli/modules/config/params.go

```diff
@@ -108,6 +109,15 @@ func NewCLIParams(context *cli.Context, rdwr ReadWriter) (*CLIParams, error) {
 		ecsConfig.CFNStackName = flags.CFNStackNamePrefixDefaultValue + ecsConfig.Cluster
 	}
 
+	if ecsConfig.Cluster == "" {
+		logrus.Warn("No default cluster is set. You may want to run `ecs-cli configure`.")
+	}
+
+	if ecsConfig.Region == "" {
```

Comment:
Should this stay?

Reply:
will this still log if they've specified a "--region" flag? If not, then makes sense to me to leave it. Any calls w/o a region are going to fail

Reply:
Turns out, its a little more complicated than I thought -- so I'm going to drop this.

The ecs-cli will fall back to the aws-cli's region config, which means at the best case we'd be asking for redundant configuration settings
```
# .ecs region unset, .aws region set
➜  ~ ecs ps --cluster default
WARN[0000] No default region is set. You may want to run `ecs-cli configure`. 
```

However, this code path isn't executed if both regions are unset. (It fails fast)
```
# .ecs and .aws region unset
➜  ~ ecs ps --cluster default
ERRO[0000] this be wehre da really bad happen           
FATA[0000] Error executing 'ps Set a region using ecs-cli configure command with the --region flag or AWS_REGION environment variable or --profile flag
➜  ~ ecs pull myrepo
FATA[0000] Error executing 'pull': Set a region using ecs-cli configure command with the --region flag or AWS_REGION environment variable or --profile flag
```